configure.ac

```diff
@@ -409,7 +409,7 @@ AC_ARG_WITH(poco,
   AS_HELP_STRING([--with-poco=DIR],[location of Poco]),
 [
 	CPPFLAGS="$CPPFLAGS -I${withval}/include"
-	LDFLAGS="$LDFLAGS -L${withval}/lib"
+	LDFLAGS="$LDFLAGS -L${withval}OD/lib"
```

Comment:
Not sure why this change is here - what does OD do? ${withval} is a directory name